src/ring_theory/integral_closure.lean

```diff
@@ -132,17 +137,22 @@ end
 @[simp]
 theorem is_integral_alg_equiv {A B : Type*} [ring A] [ring B] [algebra R A] [algebra R B]
   (f : A ≃ₐ[R] B) {x : A} : is_integral R (f x) ↔ is_integral R x :=
```

Comment:
Should we also change this to use `[alg_equiv_class]` and rename to `map_is_integral_iff` for consistency? It's only used in [one file](https://github.com/leanprover-community/mathlib/blob/ced11136c4fac7d613ae435562d8329e505ec325/src/ring_theory/integrally_closed.lean). `is_integral_alg_hom` is used in three files and I think you've fixed all of them; now the error is about `map_is_integral_int`.

Reply:
I am not sure how to do it: currently there is no way of speaking of `f.symm` if `f : F` with `[alg_equiv_class F R A B]` (since a priori `F` can be not closed under taking the inverse).

Reply:
I think we are missing a coercion from `F` to `alg_equiv`, I remember a recent discussion about this somewhere.

Reply:
Hmm yes and we are also lacking the coercion from alg_hom_class to alg_hom, which seems to block me from generalizing `is_integral_alg_hom_iff` to `[alg_hom_class]` as well.

I think this PR is good to go then!